test/nn/test_lazy_modules.py

```diff
@@ -605,10 +605,6 @@ def run_step(module, optim):
             module.register_parameter('test_param', UninitializedParameter())
             if optim_cls is torch.optim.SGD:
                 optim = optim_cls(module.parameters(), lr=0.0)
-            elif optim_cls is torch.optim.Adagrad:
```

Comment:
Note to reviewer:

As mentioned below, as we now lazily initialize step, this previous error in test/nn/test_lazy_modules.py for lazy initialization is now avoided.